app/models/slate.py

```diff
@@ -21,6 +21,13 @@ class SlateModel(BaseModel):
     description: str = None
     recommendations: List[RecommendationModel] = None
 
+    __QA_USER_IDS = ['47372502']
+    __QA_SLATE_MAP = {
+        "2e3ddc90-8def-46d7-b85f-da7525c66fb1": "9dc26792-10ed-4fbe-a13d-6cce3a89b0a1",
+        "0c09627b-a409-4768-b87d-7e1d29259785": "e8251442-ef97-422f-ad65-0f28e6f7a0d6",
+        "0f322865-64e6-472d-8147-b3d6637a7d67": "b70d65c6-9171-40bf-bddb-5a60d42dd03f",
+    }
```

Comment:
even though it's temporary code, i think some comments here would be helpful - e.g. commenting the user/email associated with that id, and commenting each slate in the map (most read, personal finance, etc).

Reply:
For future reference, the QA user details are here: https://pocket.slack.com/archives/C03QVL4SU/p1627576490339900